v2/getting-started/composer.md

```diff
@@ -1,6 +1,51 @@
 # Composer: the what, why, and how
 
-@todo
+## What is Composer?
+**[Composer](https://getcomposer.org/)** is the de facto dependency management tool for PHP. Composer allows PHP developers to easily build standalone distributable libraries that can be shared and integrated by others. This is possible in part by the PHP Framework Interoperability Group (FIG) and [PSR-4](http://www.php-fig.org/psr/psr-4/) for autoloading of class files. 
 
-* Port contents of https://glamanate.com/blog/managing-your-drupal-project-composer
-* And https://docs.google.com/presentation/d/1PK9q2dBkGHfyEO76bEVpqS61wTgA0LGbru2PECiwUnk/edit?usp=sharing
\ No newline at end of file
+[PEAR](http://pear.php.net/) is another management tool, however it installs PHP libraries in a global context via `include` and `require`. Composer supports both per-project and global usages. PEAR also does not handle dependency management.
```

Comment:
Nobody remembers PEAR, at this point it's just confusing to bring it up. Most Drupal people have used a similar tool: Drush. That's the comparison we need to make.  Composer is a Drush that can download dependencies of dependencies (Commerce needs Address? Okay, I'll download it!) and where you can specify version constraints (Download 1.x but newer than 1.3)